en/docs/get-started/about-this-release.md

```diff
@@ -12,227 +12,58 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
 
-??? note "Develop an integration from a managed API"
+??? note "Configuring API Lifecycle through Admin Portal"
+    APIs created in WSO2 API Manager have their own life cycle consisting of the following: a set of life cycle states, specific actions for each state transition, and a checklist of items before a state transition occurs. An API has a predefined life cycle consisting of six states by default. This feature allows you to customize and/or add new states, migrate API Lifecycle through the **Advance Configuration** option or through editing the `tenant-conf.json` file.
 
-    With the API Manager 4.1.0 release, you can create an integration using a managed API and configure the integration service endpoint to the managed API. This is the top-down approach in API first integration.
+??? note "Enable logs without server restart"
+    In this new version of WSO2 API Manager, you can enable or disable correlation logs as needed for troubleshooting at runtime, as a server restart is not required.
 
-    **[Learn more]({{base_path}}/tutorials/develop-an-integration-with-a-managed-api/)**
+??? note "Removal of Jaggery Apps from Portals"
+    Jaggery is not currently in an active stage of development or upkeep. Therefore, it is essential to switch to a different language which is closer to server-side while still using the react component as the client-side handler.
 
-??? note "Publishing third-party APIs on the Developer Portal"
+??? note "Registry artifact view support for MI Management API/Dashboard"
+    A new page has been added to the MI-Dashboard to view the `<MI-HOME>/registry` directory and registry metadata, properties and content.
 
-    WSO2 API Manager provides the capability to publish APIs that are deployed in an external Gateway. Since these APIs are not deployed in the WSO2 API Gateway, the API will not be proxied through the Gateway.
+    REST API resources have been added to the Micro Integrator as a **management API** to facilitate the functionalities of fetching existing registry data, adding new registry artifacts, modifying and deleting existing registries.
 
-    **[Learn more]({{base_path}}/deploy-and-publish/publish-on-dev-portal/third-party-api-support/)**
+??? note "Gateway Support for Single Usage Token"
+    The JWT access tokens supplied by the gateway can be used to access the dedicated API resource as many times as the user requires. However, if the application developer wishes to limit this functionality by allowing just one chance to access the resource, One Time Token can be utilized. A One Time Token (JWT) is a JWT that will be revoked after a single use. This functionality is only accessible for JWT access tokens.
 
-??? note "A read-only mode for API Publisher"
+??? note "Import/Export of Operation Policies with APICTL"
```

Comment:
Let's rename the title to "Support for managing Common API Policies with APICTL"

Reply:
```suggestion
??? note "Support for managing Common API Policies with APICTL"
```